brozzler/browser.py

```diff
@@ -672,7 +689,6 @@ def _try_screenshot(self, on_screenshot, full_page=False):
         """
         self.send_to_chrome(
             method="Runtime.evaluate",
-            suppress_logging=True,
```

Comment:
Nothing needs to be done in this PR, but in the long term probably a good idea to make this an option. I can look into that later.